packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -796,11 +877,29 @@ class Scaffold extends StatefulWidget {
     this.resizeToAvoidBottomPadding,
     this.resizeToAvoidBottomInset,
     this.primary = true,
+    this.extendBody = false,
     this.drawerDragStartBehavior = DragStartBehavior.down,
   }) : assert(primary != null),
+       assert(extendBody != null),
        assert(drawerDragStartBehavior != null),
        super(key: key);
 
+  /// If true and [bottomNavigationBar] or [persistentFooterButtons]
```

Comment:
Nit (with a capital N): This feels like a parenthetical statement.
```suggestion
  /// If true, and [bottomNavigationBar] or [persistentFooterButtons]
```

Reply:
Added the commas and changed "are specified" to "is specified"